lib/V8/v8-utils.cpp

```diff
@@ -3044,6 +3044,45 @@ static void JS_Read(v8::FunctionCallbackInfo<v8::Value> const& args) {
   TRI_V8_TRY_CATCH_END
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// @brief was docuBlock JS_Read
+////////////////////////////////////////////////////////////////////////////////
+
+static void JS_ReadPipe(v8::FunctionCallbackInfo<v8::Value> const& args) {
+  TRI_V8_TRY_CATCH_BEGIN(isolate)
+  v8::HandleScope scope(isolate);
+
+  if (args.Length() != 1) {
+    TRI_V8_THROW_EXCEPTION_USAGE("read(pipefd)");
+  }
+
+  int64_t f = TRI_ObjectToInt64(isolate, args[0]);
+  if (f == 0) {
+    TRI_V8_THROW_TYPE_ERROR("<filename> a file pointer");
```

Comment:
error message is not really understandable, and uses a different argument name (`filename`) than in options description (`pipefd`).